src/components/Icon/components.tsx

```diff
@@ -21,3 +21,6 @@ export { ReactComponent as Globe } from './assets/Globe.svg';
 export { ReactComponent as CaretRight } from './assets/CaretRight.svg';
 export { ReactComponent as Cross } from './assets/Cross.svg';
 export { ReactComponent as Edit } from './assets/Edit.svg';
+export { ReactComponent as DialogSuccess } from './assets/dialogSuccess.svg';
```

Comment:
Call them `Success`, `Warning` and `Danger` only, without the `Dialog` prefix.

Also, make sure that the files have exactly the same name as the components.